odoo/addons/base/models/res_partner.py

```diff
@@ -133,7 +133,8 @@ class Partner(models.Model):
     _description = 'Contact'
     _inherit = ['format.address.mixin', 'avatar.mixin']
     _name = "res.partner"
-    _order = "display_name"
+    _order = "display_name ASC, id DESC"
+    _allow_sudo_commands = False
```

Comment:
[change] you change here the order, this should not be in the commits. Therefore I propose to not change the behaviour for the customer

Reply:
is ascending not the default behaviour? This should just mean that if they ever have the same display_name, they will be sorted by id, instead of being undefined behaviour, no?

Reply:
Should be fixed, gonna deploy to test too